### PR TITLE
[release 4.7] Bug 1939553: Support binary file type secret entries

### PR DESCRIPTION
**Pull request by jcaianirh**
This is a manual cherry-pick of https://github.com/openshift/console/pull/7399

### CODE REVIEW

**Comment by yapei** [comment]
tested on a 4.7 cluster created by cluster-bot, verified the binary data is not shown on the page and console has a `Save` button to download it

File: frontend/public/components/utils/file-input.tsx [hunk 1]
```diff
@@ -4,13 +4,23 @@ import { NativeTypes } from 'react-dnd-html5-backend';
 import { DropTarget } from 'react-dnd';
 import { ConnectDropTarget, DropTargetMonitor } from 'react-dnd/lib/interfaces';
 
+import { Alert } from '@patternfly/react-core';
+/* eslint-disable-next-line */
+import { withTranslation, WithTranslation } from 'react-i18next';
```
Comment (zherman0) [inline frontend/public/components/utils/file-input.tsx:9]: This probably does not need to be duplicated

**Review by zherman0 — APPROVED** [review]

**Review by jhadvig — APPROVED** [review]
/hold cancel

**Comment by mfojtik** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.


* Score: *0.20*
* Reason: *maximum total for this z-stream is 10*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Pull request merged**